proto/src/determined/api/v1/trial.proto

```diff
@@ -376,3 +376,21 @@ message ReportTrialCheckpointMetadataRequest {
 }
 // Response to ReportTrialCheckpointMetadataRequest
 message ReportTrialCheckpointMetadataResponse {}
+
+// Gathers the rendezvous info for the trial.
+message GetTrialRendezvousInfoRequest {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "trial_id", "rendezvous_info" ] }
+  };
+  // The id of the trial.
+  int32 trial_id = 1;
+  // The id of the container. Used to verify all containers are connected.
+  string container_id = 2;
+  // The rendezvous info.
+  determined.trial.v1.RendezvousInfo rendezvous_info = 3;
+}
+// Response to GetTrialRendezvousInfoRequest
+message GetTrialRendezvousInfoResponse {
+  // The rendezvous addresses.
+  repeated string addresses = 1;
```

Comment:
This should use `determined.trial.v1.RendezvousInfo` struct?

Reply:
Yeah, I had that struct around before I rebased onto rb's trim rendezvous info pr. I suppose I could still use it, just meant to remove though.

Reply:
done